src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -644,6 +644,18 @@ bool IsMainLauncher (XElement intentFilter)
 					intentFilter.Elements (entry.Key).Any (e => ((string) e.Attribute (attName) == entry.Value)));
 		}
 
+		/// <summary>
+		/// Returns true if //application/@android:extractNativeLibs is set to false. If android:extractNativeLibs is omitted, returns false.
+		/// </summary>
+		public bool IsEmbeddedDSOsEnabled ()
```

Comment:
Is this `IsEmbeddedDSOsEnabled` name terrible? Suggestions welcome.

Reply:
It's not terrible; it's not great.  I have no suggestions.

Fortunately it isn't public API...

Reply:
should we not be using a variation of `extractNativeLibs` since that is the attribute we are reading?

Reply:
👍 yeah since this is the `ManifestDocument` class, I should name it similar to the XML attribute. `ExtractNativeLibraries()` seems a little better, and I need to make the return value opposite of what it is currently.